lib/state/actions/pokemon_actions.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:pokedex/state/app_state.dart';
+
+/// This action will be used to generate 20 Pokemon at a time.
```

Comment:
this will later be used for getting pokemons right? perhaps you could still simply this by still having a todo comment of => `/// todo: documentation will be changed later -> Get paginated pokemons per 20 items`